UniversityApp/DTOs/StudentSearchParametersDto.cs

```diff
@@ -13,9 +13,9 @@ public class StudentSearchParametersDto
         public int? Age { get; set; }
```

Comment:
This could also be moved to use FluentValidation. It wll be consistent with other parts of the application.